packages/cw1155/README.md

```diff
@@ -0,0 +1,63 @@
+# CW1155 Spec: Multiple Tokens
+
+CW1155 is a specification for managing multiple tokens based on CosmWasm.
+The name and design is based on Ethereum's ERC1155 standard.
+
+Design decisions:
+
+- Fungible tokens and non-fungible tokens are treated equally, non-fungible tokens just have one max supply.
+
+- Approval is set or unset to some operator over entire set of tokens. (More nuanced control is defined in [ERC1761](https://eips.ethereum.org/EIPS/eip-1761), do we want to merge them together?)
+
+- Metadata and token enumeration should be done by indexing events off-chain.
```

Comment:
1. token enumeration should be done by indexing events off-chain

This is because it is impossible in Ethereum. We can do better. How about using enumeration like in cw721 on tokens or tokens/owner? or cw20 on allowances?

Reply:
2. Off-chain metadata via events is okay. Adding one URL there or such would be a simple addition that seems like it is used various places.

Also, the "indexing events off-chain" should reference an example and some such indexer (I think you guys have built one, so maybe a nice place to link to it and say you could build on this)

Reply:
I added metadata and enumerable extensions like cw721, it's indeed more convenient for client I think.